app/views/posts/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Post => <%= @post.title %> by <%= @post.author.name %> </h1>
```

Comment:
- [ ] You have done well adding the posts show page. However, you still need to add the posts index page.